### ISSUE TITLE
Possible bug in cogdl/tasks/node_classification.py

### ISSUE THREAD

**Issue by xssstory** [THUDM/cogdl#114]
cogdl/tasks/node_classification.py, line 93 may be wrong? it seems that the missing_rate is not used.

```
if args.missing_rate >= 0:
    if args.model == "sgcpn":
        assert args.dataset in ["cora", "citeseer", "pubmed"]
        dataset.data = preprocess_data_sgcpn(dataset.data, normalize_feature=True, missing_rate=0)
        adj_slice = torch.tensor(dataset.data.adj.size())
        adj_slice[0] = 0
        dataset.slices["adj"] = adj_slice
```

**Comment by THINK2TRY** [comment]
@xssstory Thanks for your attention to CogDL. It does seem to be a redundant piece. We will remove that later.  Thanks for your suggestion.

**Comment by cenyk1230** [comment]
@xssstory It should pass the `args.missing_rate` to the `preprocess_data_sgcpn` function. We will fix it soon.

**Comment by cenyk1230** [comment]
@xssstory This issue is fixed in #157. Many thanks.
